guides/common/assembly_configuring-external-authentication.adoc

```diff
@@ -19,6 +19,7 @@ ifdef::satellite[]
 :!keycloak:
 endif::[]
 
+ifndef::satellite[]
 // The following ifdef sets :keycloak: to `Red Hat build of Keycloak` for Satellite builds only. For, non-Satellite builds, :keycloak: stays set to `Keycloak`.
```

Comment:
I think you could drop lines 23-27 + 33-36.

Reply:
I could, but it's possible we will want to re-include lines 22-37 for Satellite which would be a lot easier if those extra conditionals stayed in place. This is all only temporary until we can safely remove anything related to the Wildfly distribution of Keycloak.